page_permissions.ini

```diff
@@ -0,0 +1,39 @@
+; ALLOW * ==> allow all connections (including unauthenticated)
+; ALLOW auth  ==> allows all authenticated connections
+
+[permissions]
+
+; General
+/ = ALLOW auth
+/error404 = ALLOW *
+/login = ALLOW *
+/logout = ALLOW auth
+/dashboard = ALLOW auth
+/back = ALLOW auth
+
+; Participant Search
+/agency-requests = ALLOW Superuser Administrator
+/participant-search = ALLOW Superuser Administrator
+/participant-search/* = ALLOW Superuser Administrator
+/view-participant/* = ALLOW Superuser Administrator
+
+; classes
```

Comment:
Does this allow the User role to access these pages? Is that desired?

Reply:
Yes it would. Since we don't currently know what permissions are for each page, this is more of an example to show how permissions could be set.